README.md

```diff
@@ -3752,6 +3752,8 @@ Value (string):
 ### Preferences
 Set and lock preferences.
 
+**NOTE** In order to use this policy, you must clear all settings in the old **Preferences (Deprecated)** section.
```

Comment:
Can you make this GPO/Windows specific? In order to use this policy on Windows.

Thanks!